### DISCUSSION TITLE
What about bot detection? Is it reliable?

### DISCUSSION THREAD

**Discussion by rohitnishad613** [zlurad/serp-parser#89] [Q&A] [answered]
I have a question.
other SERP scraping libs have the option to set proxy and things like that.

How this lib is reliable? I mean when I search a lot of thing in a very limited period of time google give a capture (or small ban) something 

My actual question is I want to use this lib it enterprise application is it reliable?

**Comment by badluck13** [comment, marked as answer]
Hi, @rohitnishad613 this is not a scraper lib, just a parser for HTML or results page. So it's not usable for your use case

**Reply by rohitnishad613** [reply]
My bad, I do not understand it properly :(
